frontend/www/js/omegaup/components/contest/Reportv2.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <b-card
+    class="mb-5"
+    :header="ui.formatString(T.contestReport, { contest_alias: contestAlias })"
+  >
+    <b-card-text class="text-right">
+      <a :href="`/contest/${contestAlias}/report/print/`">
+        <b-icon-printer></b-icon-printer>
+      </a>
+    </b-card-text>
+    <b-card-text
+      v-for="contestantData in contestReport"
+      :key="contestantData.username"
+      class="pb-2"
+    >
+      <h1 class="text-center">{{ username(contestantData.username) }}</h1>
+      <h3 class="text-center">{{ totalPoints(contestantData) }}</h3>
+      <div
+        v-for="item in contestantData.problems"
+        :key="`${contestantData.username}_${item.alias}`"
+        class="pb-2"
+      >
+        <h3>{{ T.wordsProblem }}: {{ item.alias }}</h3>
+        <h3>{{ T.wordsPoints }}: {{ item.points }}</h3>
+        <div v-if="item.run_details">
+          <b-table
+            striped
+            hover
+            :items="getGroupsByProblemAndUser(item.run_details.details.groups)"
+            :fields="groupColumns"
+          >
+            <template #cell(details)="row">
+              <b-button
+                :title="T.wordsDetails"
+                variant="link"
+                @click="row.toggleDetails"
+              >
+                <b-icon-caret-down-square
+                  v-if="row.detailsShowing"
+                ></b-icon-caret-down-square>
+                <b-icon-caret-up-square v-else></b-icon-caret-up-square>
+              </b-button>
+            </template>
+            <template #row-details="row">
+              <b-table
+                responsive
+                striped
+                hover
+                :items="row.item.details"
+                :fields="casesColumns"
+              ></b-table>
+            </template>
+          </b-table>
+        </div>
+      </div>
+      <hr />
+    </b-card-text>
+  </b-card>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { types } from '../../api_types';
+import T from '../../lang';
+import * as ui from '../../ui';
+
+// Import Bootstrap an BootstrapVue CSS files (order is important)
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap-vue/dist/bootstrap-vue.css';
+
+// Import Only Required Plugins
+import {
+  TablePlugin,
+  ButtonPlugin,
+  CardPlugin,
+  BIconCaretDownSquare,
+  BIconCaretUpSquare,
+  BIconPrinter,
+} from 'bootstrap-vue';
+Vue.use(TablePlugin);
+Vue.use(ButtonPlugin);
+Vue.use(CardPlugin);
+
+interface GroupDetails {
+  name: string;
+  time: string;
+  wall_time: string;
```

Comment:
```suggestion
  wallTime: string;
```

como igual tenemos que convertir el objeto para cambiar los tipos, no hay nada que nos prevenga usar camelCase.

Reply:
Listo